include/api.php

```diff
@@ -122,15 +121,17 @@ function pll_home_url( $lang = '' ) {
 }
 
 /**
- * Registers a string for translation in the "strings translation" panel
+ * Registers a string for translation in the "strings translation" panel.
  *
  * @api
  * @since 0.6
  *
- * @param string $name      a unique name for the string
- * @param string $string    the string to register
- * @param string $context   optional the group in which the string is registered, defaults to 'polylang'
- * @param bool   $multiline optional whether the string table should display a multiline textarea or a single line input, defaults to single line
+ * @param string $name      A unique name for the string.
+ * @param string $string    The string to register.
+ * @param string $context   Optional, the group in which the string is registered, defaults to 'polylang'.
+ * @param bool   $multiline Optional, true if the string table should display a multiline textarea,
```

Comment:
Don't we use only multiline textareas now?

Reply:
In `PLL_Translated_Option` yes. But this function still offers the possibility to use a single line input.